source/retryable-writes/tests/etc/templates/handshakeError.yml.template

```diff
@@ -51,6 +51,10 @@ tests:
   #   - Tests whether operation successfully retries the handshake and succeeds.
 {% for operation in operations %}
   - description: "{{operation.operation_name}} succeeds after retryable handshake network error"
+    {%- if operation.operation_name == 'createChangeStream' %}
+    runOnRequirements:
+      - serverless: forbid
+    {%- endif %}
```

Comment:
Was this just done for consistency? It seems unlikely that `createChangeStream` would ever be tested for retryable writes.

Reply:
> It seems unlikely

Note; this was also why the retryable writes template file does not contain other change stream specific behavior such as `saveResultAsEntity`.

Reply:
Oh I did this without thinking. Removed.